common/views/components/styled/SpacingComponent.tsx

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 
-const SpacingComponent = styled.div.attrs({
-  className: 'spacing-component',
-})`
+const SpacingComponent = styled.div.attrs<{ sliceType?: string }>(props => ({
+  className: `spacing-component ${props.sliceType ? props.sliceType : ''}`,
+  'data-slice-type': props.sliceType,
```

Comment:
I've added a class and a data-slice-type attribute. Didn't see a good reason not to. We could also add a class/data-slice-weight attribute if/when we need it

Reply:
I'd add a comment here so we know what it was for if we end up never using it?

Reply:
I got rid of it since it's not currently in use. Would be simple to add if/when it's needed, but no point in having it kicking around just in case?